setup.py

```diff
@@ -41,9 +41,9 @@ def read_requirements(filename):
 
 requirements = {
     "base": read_requirements("requirements/base.txt"),
+    "merlin": read_requirements("requirements/merlin.txt"),
```

Comment:
why do we need an extra named `merlin` if this is going to be part of `install_requires`?

Reply:
This is to make it easier in CI/CD to install only the base-requirements & install all merlin-deps from the main branches.

Reply:
Supporting the tox environment with installation from branches makes sense,

However, I wonder if we would like to restrict the keys we publish in `extras_requires` to only those that we expect users of the pacakge to use, since that becomes part of the interface to the package.

Reply:
Ah that's a good point. So that would imply changing it to: 

```python
"base": read_requirements("requirements/base.txt") + read_requirements("requirements/merlin.txt")
```

In that case maybe `base_external.txt` & `base_merlin.txt` might be better names?

Reply:
yes I think that change would be clearer, both to setup.py and the rename of the requirements files